Sources/MapboxNavigation/Resources/en.lproj/Localizable.strings

```diff
@@ -229,6 +238,18 @@
 /* Specific route feedback that route contained impassible, narrow roads. */
 "ROUTE_QUALITY_ROADS_TOO_NARROW_FEEDBACK" = "Route had roads too narrow to pass";
 
+/* Alternatives selection note about equal travel distance. */
+"SAME_DISTANCE" = "Same distance";
+
+/* Alternatives selection note about equal travel time. */
+"SAME_TIME" = "Same time";
```

Comment:
Should we use Title Case or Sentence case in CarPlay list item labels? If sentence case, the slash you originally used might be better, to avoid “Same distance, Same time”, which would look like a typo.